bin/clover/src/pipeline-steps/createPolicyDocumentInputSockets.ts

```diff
@@ -0,0 +1,49 @@
+import { setAnnotationOnSocket } from "../spec/sockets.ts";
```

Comment:
I wonder if this logic should happen in `assetSpecificOverrides`. I had some intention to gather those in one place to make it more obvious where we were special casing things

Reply:
I didn't put them there because it isn't asset-specific--this feels like an awsSpecificHeuristic to me, maybe?

Reply:
(I think it's something we can probably move after we merge, too--I'm not sure there's a clear answer on where to move it if anywhere)